com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -773,25 +773,23 @@ private void ExtractNetworkMetricsFromPipeline(NetworkPipeline pipeline, Network
 
         private int ExtractRtt(NetworkConnection networkConnection)
         {
-            if (NetworkManager.IsServer)
+            if (m_Driver.GetConnectionState(networkConnection) != NetworkConnection.State.Connected)
```

Comment:
Detail but why is the `NetworkManager.IsServer` check out of the method instead of adding it to the if(m_Driver.GetConnectionState(networkConnection) != NetworkConnection.State.Connected) ?

Reply:
Because we want `ExtractRtt` to work on servers too. Metrics want 0 for the RTT on servers, but `GetCurrentRtt` must work on servers too. So moving the check out of the method allows `ExtractRtt` to be used more widely.